packages/venia-ui/lib/components/Main/main.css

```diff
@@ -10,9 +10,10 @@
 }
 
 .page {
+    display: grid;
```

Comment:
Causes issues on mobile for the CMS page

[![Image from Gyazo](https://i.gyazo.com/481d090a5de66b0f1cc891e35559ebb6.gif)](https://gyazo.com/481d090a5de66b0f1cc891e35559ebb6)